Bottom: c74e388c572cbb004e4db0612b974b9f6c3c7e5c
Top:    120ac798368c116b3dd712159ccf91bbf3ff3325
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-08 23:53:42 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 43dee33..b79579e 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2746,7 +2746,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
